mesonbuild/cmake/traceparser.py

```diff
@@ -660,6 +660,7 @@ def _guess_files(self, broken_list: T.List[str]) -> T.List[str]:
 
         fixed_list = []  # type: T.List[str]
         curr_str = None  # type: T.Optional[str]
+        path_found = False # type: bool
```

Comment:
`path_found` currently isn't reset in the other branches (`elif os.path.isfile(curr_str)`, etc.).